test/regression/aarch64/MicroOperation.cc

```diff
@@ -10,6 +10,7 @@
 namespace {
 
 using MicroOp = AArch64RegressionTest;
+using namespace simeng::arch::aarch64;
```

Comment:
Could add `InstructionGroups` to namespace so this doesn't need to be repeatedly qualified in the tests. Thoughts @UoB-HPC/simeng-developers ?

Reply:
Doing this makes development much easier so done in #420